robot/Cumulus/tests/browser/recurring_donations/zcreate_open_erd_for_contact.robot

```diff
@@ -0,0 +1,60 @@
+*** Settings ***
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/RecurringDonationsPageObject.py
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             Enable RD2
+Suite Teardown  Delete Records and Close Browser
+
+
+***Keywords***
+# Setup a contact with parameters specified
+Setup Test Data
+    Setupdata           contact               contact_data=${contact_fields}
+
+*** Variables ***
+&{contact_fields}  Email=testuser@example.com
+
+${frequency}  1
+${day_of_month}  2
+${amount}  100
+${method}  Credit Card
```

Comment:
Can you declare the global variables (in your Setup and Variables section) in CAPS and also update accordingly in your test case? 
Sorry for being persistent. As someone who doesn't work on NPSP, when I look at the test case, it's hard for me to figure out where a certain variable is defined. Seeing it in CAPS will give the clarity that it is defined outside of the test.

Reply:
Thanks Safiya. Good one. I updated